### PR TITLE
[exporter/loadbalancing] fix duplicate component IDs of OTLP exporters

### PR DESCRIPTION
**Pull request by leizhag**
**Description:** Component IDs of OTLP exporters maintained by load balancing exporter are [supposed to be unique](https://github.com/open-telemetry/opentelemetry-collector/blob/6fb884b2dbdc37ef2e1aea924040822ce38584bd/config/identifiable.go#L37), but currently they're always "otlp". This leads to missing metrics of exporter helper, which is used by the OTLP exporter.

Before fix:
```
otelcol_exporter_queue_size{exporter="otlp",service_instance_id="479b7ec6-c35b-42f3-877a-002be5abc337",service_version="latest"} 0
```

After fix:
```
otelcol_exporter_queue_size{exporter="otlp/otel-collector-1:4317",service_instance_id="479b7ec6-c35b-42f3-877a-002be5abc337",service_version="latest"} 0
otelcol_exporter_queue_size{exporter="otlp/otel-collector-2:4317",service_instance_id="479b7ec6-c35b-42f3-877a-002be5abc337",service_version="latest"} 0
otelcol_exporter_queue_size{exporter="otlp/otel-collector-3:4317",service_instance_id="479b7ec6-c35b-42f3-877a-002be5abc337",service_version="latest"} 0
```

### CODE REVIEW

File: exporter/loadbalancingexporter/example/docker-compose.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ services:
       - "8889"   # Prometheus exporter metrics
       - "13133" # health_check extension
       - "4317"        # OTLP gRPC receiver
-      - "55670" # zpages extension
+      - "55679" # zpages extension
```
Comment (codeboten) [inline exporter/loadbalancingexporter/example/docker-compose.yaml:15]: Is this change related?

Comment (leizhag) [reply exporter/loadbalancingexporter/example/docker-compose.yaml:15]: Not quite. Found this typo when testing the fix with Docker compose. Should I fire another PR for this?

Comment (TylerHelmuth) [reply exporter/loadbalancingexporter/example/docker-compose.yaml:15]: Ya

Comment (leizhag) [reply exporter/loadbalancingexporter/example/docker-compose.yaml]: Unrelated typo fixing was removed from this PR.

**Review by codeboten — COMMENTED** [review]
Please add an entry to the changelog, otherwise this looks good 👍🏻

**Comment by jpkrohling** [comment]
What's the state of this PR?
